lib/pbench/cli/getconf.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/python3
```

Comment:
Why do we still have `getconf.py`?

Reply:
Never mind, this is the module that `pbench-config` generated command actually uses.